doc/content/source/auxkernels/SkinnedBins.md

```diff
@@ -0,0 +1,36 @@
+# SkinnedBins
+
+!syntax description /AuxKernels/SkinnedBins
+
+## Description
+
+Displays how each element in the `[Mesh]` maps to a skinned bin distribution
+(created by a [MoabSkinner](/userobjects/MoabSkinner.md)). You can either
+view the entire individualized bin distribution with `skin_by = all` (the default),
+or inspect how elements map to the binning in temperature, density, and subdomain
+by setting `skin_by` to `temperature`, `density`, or `subdomain`, respectively.
```

Comment:
If we skin the tet mesh by temperature only, the temperature regions may span multiple subdomains (i.e. materials). In that case, how are materials assigned in the DAGMC model?

Reply:
This comment here for `SkinnedBins` only refers to visualizing the bins (so you can separately look at the temperature bins, the density bins, and the subdomain bins -- or the convolution of them all).

In terms of the skinning operation (in `MoabSkinner`), we always skin by temperature + subdomain. So the mesh subdomains need to correspond to material boundaries in the DAGMC model. Eventually I'd like to add some error checking to enforce that, but I plan to leverage the mesh <--> cell mapping in `OpenMCCellAverageProblem` (which already has functions to map cell instances/materials to mesh elements) to do that. So I was going to save it for a future PR.

Reply:
Good question, and helpful answer.
As for equivalency with DAGMC model I would advocate for eliminating the dagmc model step entirely. I have an open branch in AURORA (unfortunately not touched since the summer) which will remove the need for it.

Reply:
Nice! So you mean that the initial DAGMC model would be built straight from the MOOSE input files, with no need for a .h5m file?

Reply:
That will be a really nice feature. For now, this PR doesn't make any assumptions about OpenMC, since the skinning "operation" is separated. So we can certainly incorporate those developments in a later PR (without conflicting with anything here, is what I'm meaning to emphasize).